Makefile

```diff
@@ -156,7 +156,6 @@ static: install-tools
 	@revive -formatter friendly -config revive.toml $(PACKAGES)
 
 tidy:
-	@echo "go mod tidy"
```

Comment:
Why remove it?

Reply:
The cmd `go mod tidy` itself will be outputted if there is no `@` at the beginning. So the `echo` is unncessary.